### PR TITLE
docs: fixes to links in IaC and FAQ

### PR DESCRIPTION
**Pull request by robbyandrews**

### CODE REVIEW

File: content/how-to/cli/using-cli/index.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ In addition to .blubracket-ignore file, it is possible to mark a secret to be ig
 
 Ignoring whole folders and/or secret type all the time could be dangerous as it can be easy to miss secrets. The recommended ways to deal with false positives are inline comments or ignoring the particular secret using `secret_value` rule in the `.blubracket-ignore` file.
 
-##### Adding Checks for your Own Secrets
+#### Adding Checks for your Own Secrets {#own-secrets}
```
Comment (misterbisson) [inline content/how-to/cli/using-cli/index.md:91]: Sentence case

**Review by misterbisson — APPROVED** [review]
Merge this, but let's get back to fixing the casing problems in the CLI page?

File: content/how-to/cli/using-cli/index.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ resources:
 
 [How do I Install the BluBracket Command Line Interface (CLI) Tool?](/how-to/cli/installation/)
 
-#### Setting Git Hooks
+#### Setting Git hooks
```
Comment (misterbisson) [inline content/how-to/cli/using-cli/index.md:17]: Git is not a proper noun

**Review by misterbisson — DISMISSED** [review]
Merge this, but let's get back to fixing the casing problems in the CLI page?

File: content/how-to/cli/using-cli/index.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ resources:
 
 [How do I Install the BluBracket Command Line Interface (CLI) Tool?](/how-to/cli/installation/)
```
Comment (misterbisson) [inline content/how-to/cli/using-cli/index.md:15]: Sentence case

File: content/how-to/cli/using-cli/index.md [hunk 4]
```diff
@@ -74,21 +74,21 @@ secret_type:password.*
 secret_value:my_test_gcp_token
 ```
 
-##### Current Limitations:
+##### Current limitations:
   
 * negate mask (!) is not supported
 
 * .blubraket-ignore is read only from the root repo directory, placing it in a sub directory will have no effect.
```
Comment (misterbisson) [inline content/how-to/cli/using-cli/index.md:81]: Let's put backtics around `.blubraket-ignore`, maybe?

File: content/how-to/cli/using-cli/index.md (hunk 2 above)
Comment (robbyandrews) [reply content/how-to/cli/using-cli/index.md]: Fixed in new commit

File: content/how-to/cli/using-cli/index.md (hunk 3 above)
Comment (robbyandrews) [reply content/how-to/cli/using-cli/index.md]: fixed in new commit

File: content/how-to/cli/using-cli/index.md (hunk 4 above)
Comment (robbyandrews) [reply content/how-to/cli/using-cli/index.md]: added backticks

**Review by misterbisson — APPROVED** [review]

**Pull request merged**